rest-api-spec/src/main/resources/rest-api-spec/api/ml.put_trained_model_definition_part.json

```
@@ -0,0 +1,38 @@
+{
+  "ml.put_trained_model_definition_part":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/put-trained-model-definition-part.html",
```

That URL will cause problems, since "current" is 7.14 at the moment. It's better to leave it unset, or set to something more generic for now:
```suggestion
      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/ml-df-analytics-apis.html",
```